server/db/migrate/20160402130519_add_osp_fields_to_deployments.rb

```diff
@@ -0,0 +1,23 @@
+class AddOspFieldsToDeployments < ActiveRecord::Migration
```

Comment:
+1 looks ok. Now I see why this should be broken up. Deployment is getting gnarly